### PR TITLE
Chainlink Contracts v1 update

### PR DESCRIPTION
**Pull request by dwightjl**
- Add the v1.0.0 @chainlink contracts package
- Update examples to use v0.8 contracts
- Update URLs to use the master branch
- Add the v0.8 LinkToken.json ABI to the Docs repo because ABIs were removed from the contracts package

### CODE REVIEW

File: src/scripts/link-to-wallet.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { MetaMaskInpageProvider } from "@metamask/providers"
 import detectEthereumProvider from "@metamask/detect-provider"
 import { BigNumberish, ethers } from "ethers"
 import { Web3Provider } from "@ethersproject/providers"
-import LinkToken from "@chainlink/contracts/abi/v0.4/LinkToken.json"
```
Comment (aelmanaa) [inline src/scripts/link-to-wallet.ts:5]: Hi @snehaagni 

the complied ABIs are not part of the npm package anymore?

**Comment by aelmanaa** [comment]
@dwightjl let's not forget this [PR](https://github.com/smartcontractkit/documentation/pull/1763)

File: public/samples/Automation/example.sol [hunk 2]
```diff
@@ -1,9 +1,9 @@
 // SPDX-License-Identifier: MIT
-pragma solidity ^0.7.0;
+pragma solidity ^0.8.0;
 
 // KeeperCompatible.sol imports the functions from both ./KeeperBase.sol and
 // ./interfaces/KeeperCompatibleInterface.sol
-import "@chainlink/contracts/src/v0.7/KeeperCompatible.sol";
+import "@chainlink/contracts/src/v0.8/automation/KeeperCompatible.sol";
```
[the remark this answers is not in the document]
Comment (dwightjl) [reply public/samples/Automation/example.sol:6]: Done

File: src/scripts/link-to-wallet.ts (hunk 1 above)
Comment (dwightjl) [reply src/scripts/link-to-wallet.ts:5]: Next beta version will include them. Keeping this comment open until we restore this import.

File: public/samples/Functions/adding-functions-to-projects/FunctionsExample.sol.ignore
Comment (dwightjl) [inline public/samples/Functions/adding-functions-to-projects/FunctionsExample.sol.ignore:1]: @aelmanaa can we remove this or are we planning to use it in the future? Looks like it expects the 1_0_0 functions.

File: src/scripts/link-to-wallet.ts (hunk 1 above)
Comment (dwightjl) [reply src/scripts/link-to-wallet.ts:5]: Resolved

**Pull request merged**